developer_defaults.tf

```diff
@@ -2,12 +2,12 @@ variable "env" {
   description = "The environment you wish to use"
 }
 
-variable "aws_secret_key" {
-  description = "Amazon Web Service Secret Key"
+variable "aws_account_id" {
+  description = "Amazon Web Service Account ID"
 }
 
-variable "aws_access_key" {
-  description = "Amazon Web Service Access Key"
+variable "aws_assume_role_arn" {
```

Comment:
With instance profiles don't they automatically assume their attached role, didn't think you needed this step.  How will this work when running locally?

Reply:
Yeh so the instance profile will have permission to assume this role specific for deploying. all our dev users will also have permission to assume this role.

Reply:
But potentially you could leave this bit out and it would still work?  Devs would use their existing permissions rather than needing to assume a role and ec2 will automatically use the role of the instance profile.

Reply:
The whole idea is that there is a specific role we can lock down for deploying.
also, this makes it so we can assume a role in a different AWS account.